internal/pkg/local/paths.go

```diff
@@ -0,0 +1,158 @@
+package local
+
+import (
+	"fmt"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+)
+
+type PathsGenerator struct {
```

Comment:
2. 

Druha vec, co bolo potrebne fixnut je `config <-> config` vztah.
Doteraz sa generovanie path spoliehalo na to, ze ak idu objekty v poradi `branches, configs, rows`, tak rodic kazdeho objektu uz ma vygenerovanu path.
Pri vztahu `config <-> config` to uz ale neplati, kedze nevieme, ktory config pride ako prvy (aj ked su uz oba isto nacitane).

Preto musime generovat path rekurzivne (podobne ako pri nacitavani manifestu).

Reply:
Aby to bolo prehladnejsie, tak som vytvoril strukturu `PathsGenerator`.